ntlm.go

```diff
@@ -251,18 +251,18 @@ func getNTLMv2AndLMv2ResponsePayloads(target, username, password string, challen
 	return
 }
 
-func negotiateExtendedSessionSecurity(flags uint32, message []byte, challenge [8]byte, username, password string) (lm, nt []byte, err error) {
+func negotiateExtendedSessionSecurity(flags uint32, message []byte, challenge [8]byte, username, password, userDom string) (lm, nt []byte, err error) {
 	nonce := clientChallenge()
 
 	// Official specification: https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-nlmp/b38c36ed-2804-4868-a9ff-8dd3182128e4
 	// Unofficial walk through referenced by https://www.freetds.org/userguide/domains.htm: http://davenport.sourceforge.net/ntlm.html
 	if (flags & _NEGOTIATE_TARGET_INFO) != 0 {
-		targetInfoFields, target, err := getNTLMv2TargetInfoFields(message)
+		targetInfoFields, _, err := getNTLMv2TargetInfoFields(message)
```

Comment:
With this change, the target from this message isn't used anywhere. That may be fine. From what I can tell, you are substituting the domain in the username rather then using the ntlm derived authentication context. Is that somewhat correct?

Reply:
Oh, I forgot remove it. I'll fix it